crates/sui-json-rpc/src/read_api.rs

```diff
@@ -91,7 +93,11 @@ impl RpcReadApiServer for ReadApi {
             .state
             .get_object_read(&object_id)
             .await
-            .map_err(|e| anyhow!("{e}"))?
+            .tap_ok(|_| debug!(?object_id, "Successfully get transaction"))
```

Comment:
i don't think we should log this, we get thousands of get_object RPCs per second.

Reply:
we are already [logging](http://grafana.shared.internal.sui.io:3000/explore?orgId=1&left=%7B%22datasource%22:%22w7FEhcMVz%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bapp%3D~%5C%22fullnode%5C%22%7D%20%7C%3D%20%60getObject%7D%60%22,%22queryType%22:%22range%22,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-5m%22,%22to%22:%22now%22%7D%7D) this today, although the object_id is not logged. This is indeed going to double the log volume.

The reason I want to log this is to see what objects are being queried and what optimization we could have to that. I will remove this for now and let's see how the wallet object query fix improves things